hibernate-core/src/test/java/org/hibernate/orm/test/sql/exec/HqlDeleteExecutionTests.java

```diff
@@ -48,8 +48,8 @@ public void testSimpleDelete(SessionFactoryScope scope) {
 	@Test
 	public void testSimpleRestrictedDelete(SessionFactoryScope scope) {
 		scope.inTransaction(
-				session -> session.createQuery( "delete BasicEntity where data = :filter" )
-						.setParameter( "filter", "abc" )
+				session -> session.createQuery( "delete BasicEntity where data = :thefilter" )
```

Comment:
You have to add this token to the `identifier` rule at the end of `HqlParser.g4` to avoid the changes in this file

Reply:
Thanks, did not know this.